src/Lykke.Service.Balances.Services/RedisCacheExtensions.cs

```diff
@@ -22,7 +23,14 @@ internal static class RedisCacheExtensions
                 record = await getRecordFunc();
                 if (record != null)
                 {
-                    await cache.TryHashSetAsync(key, field, record, cacheExpiration, log);
+                    if (!await cache.KeyExistsAsync(key) && reloadAllAction != null)
```

Comment:
it is not thread-safe operation, is it ok ? I mean that several threads can start to reload